### DISCUSSION TITLE
Contribution activity is not showing under overview section

### DISCUSSION THREAD

**Discussion by dhakadrohan98** [community/community#46019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

 Contribution activity is not showing under overview section on my github profile from 29 Jan,2023 to 1 Feb,2023. Please help regarding the same why Contribution activity is not showing in my calendar & history of Contribution activity?

![Github problem](https://user-images.githubusercontent.com/48330715/216056127-aa5ed678-a18a-44b1-8b01-5752dc0616eb.png)

**Comment by SilasRodrigues19** [comment]
Hey @dhakadrohan98 

Looks like it's right, it's displaying your commits

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dhakadrohan98 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
